test/unit_test/common/test_parser_config_utils_mistral.py

```
@@ -0,0 +1,22 @@
+
+
+def test_mistral_ocr_suffix_normalized():
+    raw = "mistral-ocr-latest@inst@Mistral OCR"
+    layout, model_name = normalize_layout_recognizer(raw)
+    assert layout == "Mistral OCR"
+    assert model_name == raw
+
+
+def test_mistral_ocr_suffix_case_insensitive():
+    raw = "mistral-ocr-latest@inst@MISTRAL OCR"
+    layout, model_name = normalize_layout_recognizer(raw)
```

Ruff (0.15.21)

[warning] Unpacked variable `model_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)